net/openvpn/files/openvpn.init

```diff
@@ -159,6 +162,9 @@ openvpn_add_instance() {
 		${client:+--ipchange "/usr/libexec/openvpn-hotplug ipchange $name"} \
```

Comment:
this line not needed if you redefined new behavior below

Reply:
If we remove it the ipchange action in the hotplug script (`/etc/hotplug.d/openvpn/01-user`) is never triggered.

The beauty of the solution is that it combines OpenWRT's hotplug actions via the hotplug script but makes it also compatible with the old way of placing script events in the openvpn config file by parsing the openvpn config file and add those scripts to the hotplug actions by setting it as an environmental variable e.g. `$user_ipchange`.

By removing  `${client:+--ipchange "/usr/libexec/openvpn-hotplug ipchange $name"} \ ` we are breaking this.

Sure the ipchange script in the openvpn config file is exectued but by openvpn and not by the hotplug action as it is intended to be.

Reply:
If parameter specified more once, only last occurence will be accepted to execute. So, even without removing, hotplug event do not exeuted

Reply:
The current code already overrides the options from the config file with command-line options, so OpenVPN ignores the `route-up`, `route-pre-down`, and `ipchange` options in the config file.

The proposed code exports the relevant options as environment variables which are used in the hotplug script to call the action-specific scripts and make the options from the config file work again.